### ISSUE TITLE
Bootstrap select  property '_dataApiKeydownHandler' error

### ISSUE THREAD

**Issue by ceds** [snapappointments/bootstrap-select#2446]
I'm using version 1.13.15.

The following code:

```
<div class="form-group">
   <label for="measureId">Measure</label>
   <select class="form-control selectpicker" id="measureId" data-live-search="true" data-size="10"></select>
</div>

$('#measureId').selectpicker();
```

gives me the following error:

> bootstrap-select.js:8 Uncaught TypeError: Cannot read property '_dataApiKeydownHandler' of undefined
>     at bootstrap-select.js:8
>     at dispatch (jquery 3.3.1.js:2)
>     at y.handle (jquery 3.3.1.js:2)

The select control is not working at all.  I think it's due to the exception.  Any ideas ?

**Comment by caseyjhol** [comment]
This should be fixed in v1.13.16 (fixed with commit 6ecef4ed537de2089b1c501d51b4232be416b168). That will most likely be released later today.

**Comment by caseyjhol** [comment]
Released in [v1.13.16](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.16)!
